### ISSUE TITLE
[feature request] Password protect changing settings of app

### ISSUE THREAD

**Issue by AndreasA** [TilesOrganization/support#29]
Hi,

I want to enable some tiles for a tablet that I am limiting to certain apps, e.g. for "Point of Sale" use or similar, e.g. using restricted profiles or Acer's EZ Mode.
Everything is fine so far. However, in order for the Quick Settings Tiles from "Tiles" to work, I have to allow "Tiles" to be opened too. In order to disallow people to change settings inside "Tiles" (due to system settings restrictions they cannot do that much anyway) it would be good, if when opening the App to modify settings a password protection could be set.

**Comment by rascarlo** [comment]
Hi @AndreasA and thanks for your feedback.
Do you mind to explain what you mean with 

> in order for the Quick Settings Tiles from "Tiles" to work, I have to allow "Tiles" to be opened too

**Comment by AndreasA** [comment]
Hi,,

what I meant is the following:
In Acer's EZ Mode I can limit which Apps can be run by the tablet. If the app isn't allowed it will not be started or closed automatically.
So if I add a quick settings tile using Tiles and want the user in active EZ Mode to be able to use that tile, I have to allow Tiles to be run as otherwise the tile is visible but nothing happens, e.g. tested with the Lock tile (Permissions have been set accordingly for that tile to work in the EZ Mode account).
However, if a user can launch Tiles then the can also enable other Tiles which I actually don't want them to be able to do. So the easiest solution would be to require a Password when accessing the Tiles configuration.
I hope this explains it.

**Comment by rascarlo** [comment]
@AndreasA I belive password protecting the application settings is pointless since any user can edit the quick settings layout and remove the tiles added by the main user.

**Comment by AndreasA** [comment]
Yes but they can enable further tiles, e.g. one could activate some Tiles in the tiles app but not all of them.
As there is no password protection, they could not enable additional tiles. It isn't that big of a deal, as any tiles that change settings still require a password but...
